src/components/ActionCard.tsx

```diff
@@ -1,3 +1,4 @@
+import { StaticImageData } from "next/image"
```

Comment:
Mentioned in Discord, but is `StaticImageData` our preferred approach to typing the image `src`?